frontend/cypress/integration/admin/admin-smoke-tests.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  aliasMutation,
+  aliasQuery,
+  setUpCommonGraphqlIntercepts,
+} from "../../support/graphql-test-utils";
+import { getInputByLabel } from "../../support/helpers";
+
+describe("Admin Smoke Tests", () => {
+  const loginViaUI = (role) => {
+    cy.fixture("users.json").then((users) => {
+      const user = users[role];
+      cy.get("input[name=username]").type(user.email);
+    });
+    cy.findByText("Sign-in").click();
+  };
+
+  beforeEach(() => {
+    // common requests
+    setUpCommonGraphqlIntercepts();
+    // page specific requests
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "AllUsers");
+      aliasQuery(req, "User");
+      aliasMutation(req, "UpdateUserAsAdmin");
+    });
+
+    cy.visit("/en/admin");
+  });
+
+  it("Searches for a user, reviews the profile, and edits the phone number", () => {
+    // hit the home page, login, hit the dashboard
+    cy.contains("h1", "Home");
```

Comment:
Wondering about whether we should ever use cypress' native `contains` command, or instead the testing-library query selectors 🤔 Will leave comment in thread

```suggestion
    cy.findByRole("heading", { name: /Home/i });
```

Reply:
You're probably right that findByRole is better.  I'll have to do some experimentation.  I might have to add some roles, too, which is probably a good side-effect. :laughing:

Reply:
I've removed all the contains and switched to findBy selectors.  I agree that it's probably a better way to go.  Applied in https://github.com/GCTC-NTGC/gc-digital-talent/pull/3084/commits/383b3126d8c6c36e4ef791e4cb651144a270797b

Reply:
yeah, the little a11y tidy tasks from using the findByRole selector totally sold me on the testing-library ethos :)